bin/setup

```diff
@@ -24,16 +24,21 @@ chdir APP_ROOT do
 
   puts "\n== Copying sample files =="
   unless File.exist?('config/database.yml')
-    cp 'config/database.yml.sample', 'config/database.yml'
+    FileUtils.cp 'config/database.yml.sample', 'config/database.yml'
   end
 
   puts "\n== Preparing database =="
-  system! 'bin/rails db:setup'
+  db_exists = system('psql -o /dev/null -q -n -c "select 1" PracticalDeveloper_development')
```

Comment:
The database name could potentially be different, set by `ENV['DATABASE_URL']`

Reply:
Good point, maybe we could make it an optional argument and default to `PracticalDeveloper_development` if that's missing.

Reply:
Sure, let's do it.